components/dashboards-web-component/src/viewer/components/DashboardRenderer.jsx

```diff
@@ -178,7 +178,7 @@ export default class DashboardRenderer extends Component {
         const exportButton = document.createElement('i');
         exportButton.title = 'Generate Report';
         //TODO: change icon
-        exportButton.className = 'fw fw-document widget-report-generation-button';
+        exportButton.className = 'fw fw-download widget-report-generation-button';
```

Comment:
As discussed offline, lets use the export icon here.
http://wso2.github.io/ux-font-wso2/icons.html#export